.travis.yml

```diff
@@ -0,0 +1,58 @@
+sudo: required
+dist: trusty
+
+# install the pre-release chef-dk.  Use chef-stable-trusty to install the stable release
+addons:
+  apt:
+    sources:
+      - chef-stable-trusty
+    packages:
+      - chefdk
+
+# Don't `bundle install` which takes about 1.5 mins
+install: echo "skip bundle install"
+
+# branches:
+#   only:
+#     - master
+
+services: docker
+
+env:
+  matrix:
+    - INSTANCE=varnish3-ubuntu-1404
+    - INSTANCE=varnish3-ubuntu-1204
+    - INSTANCE=varnish3-centos-6
+    - INSTANCE=varnish4-ubuntu-1404
+    - INSTANCE=varnish4-ubuntu-1204
+    - INSTANCE=varnish4-centos-7
+    - INSTANCE=varnish4-centos-6
+    - INSTANCE=varnish41-ubuntu-1404
+    - INSTANCE=varnish41-ubuntu-1204
+    - INSTANCE=varnish41-centos-7
+    - INSTANCE=varnish41-centos-6
+    - INSTANCE=distro-ubuntu-1404
+    - INSTANCE=distro-ubuntu-1204
+    - INSTANCE=distro-centos-7
+    - INSTANCE=distro-centos-6
+    - INSTANCE=full-stack-ubuntu-1404
+    - INSTANCE=full-stack-ubuntu-1204
+    - INSTANCE=full-stack-centos-7
+    - INSTANCE=full-stack-centos-6
+
+before_script:
+  - sudo iptables -L DOCKER || ( echo "DOCKER iptables chain missing" ; sudo iptables -N DOCKER )
+  - eval "$(/opt/chefdk/bin/chef shell-init bash)"
+  - /opt/chefdk/embedded/bin/chef --version
+  - /opt/chefdk/bin/chef exec bundle install
+  - /opt/chefdk/embedded/bin/cookstyle --version
+  - /opt/chefdk/bin/chef exec foodcritic --version
+
+script:
+ - KITCHEN_LOCAL_YAML=.kitchen.docker.yml /opt/chefdk/embedded/bin/kitchen verify ${INSTANCE}
+
+matrix:
+  include:
+    - script:
+      - /opt/chefdk/bin/chef exec rake
```

Comment:
You'll want to switch this over to delivery local all since you shipped the delivery config